src/Native/Native.proj

```diff
@@ -86,6 +86,10 @@
           SourceFiles="$(NuGetPackageRoot)mlnetmkldeps\$(MlNetMklDepsPackageVersion)\runtimes\$(PackageRid)\native\$(NativeLibPrefix)MklImports$(NativeLibExtension)"
           DestinationFolder="$(NativeAssetsBuiltPath)" />
 
+    <!-- Copy MklImports pdb over -->
+    <Copy Condition="'$(OS)' == 'Windows_NT' And '$(TargetArchitecture)' != 'arm64' And '$(TargetArchitecture)' != 'arm'" SourceFiles="$(NuGetPackageRoot)mlnetmkldeps\$(MlNetMklDepsPackageVersion)\runtimes\$(PackageRid)\native\MklImports.pdb"
```

Comment:
I am seeing this condition repeated in various places. Would it make sense to move this condition to some property? 

```Xml
<PropertyGroup Condition="'$(OS)' == 'Windows_NT'">
    <NonArmOnWindows Condition="'$(TargetArchitecture)' != 'arm64' And '$(TargetArchitecture)' != 'arm'">true</NonArmOnWindows>
</PropertyGroup>
```

Reply:
That makes sense. If a property is empty will MSBUILD evaluate it as false?

Reply:
> That makes sense. If a property is empty will MSBUILD evaluate it as false?

msbuild will evaluate it to empty string (i.e. it is not set). your condition would be something like:

```
Condition="'$(NonArmOnWindows)' == 'true'"
```

so any other value of the property would be considered as false.

Reply:
Makes sense. I have made that change.